transforms/universal/pdf2md/python/.gitignore

```diff
@@ -0,0 +1,39 @@
+test-data/output
```

Comment:
Do you need to ignore HF_CACHE folder you reference in the help text for the transform?

Reply:
the huggingface cache is usually in the home folder, not inside the repo, so I don't think it is needed